packages/react-cookie/README.md

```diff
@@ -66,6 +66,17 @@ const [cookies, setCookie, removeCookie] = useCookies(['cookie-name']);
 
 Let you optionally specify a list of cookie names your component depend on or that should trigger a re-render. If unspecified, it will render on every cookie change.
 
+### `options` (optional)
+
+- options (object):
+  - doNotParse (boolean): do not convert the cookie into an object no matter what
```

Comment:
Is there any reason you documented the doNotParse option, but not the doNotUpdate option? It's also part of the same parameters.

Reply:
@eXon  sorry for the delay, I was copying documentation structure from rest of readme. seems like all followed the same of only showing `doNotParse`. I can update if you want to include doNotUpdate as well

Reply:
@presto2116 Yes please 👍

Reply:
pulled in latest changes and updated the readme. let me know if there is anything else you would need